assets/nest/multiple-choice-features/authorization/mongodb/test/auth0/jest-e2e.config.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
+module.exports = {
+  moduleFileExtensions: [
+    'js',
+    'json',
+    'ts'
+  ],
+  rootDir: '..',
+  testEnvironment: 'node',
+  testRegex: '.e2e-spec.ts$',
+  transform: {
+    '^.+\\.(t|j)s$': 'ts-jest'
+  },
+  coverageDirectory: 'coverage-e2e',
+  collectCoverageFrom: [
+    '<rootDir>/src/api/**/*.(t|j)s'
+  ],
+  coveragePathIgnorePatterns: [
+    '<rootDir>/src/main.ts$',
+    '<rootDir>/.*spec.ts$',
+    '<rootDir>/.*module.ts$',
+    '<rootDir>/src/.*/index.ts$',
+    '<rootDir>/src/.*dto.ts$',
+    '<rootDir>/src/.*entity.ts$',
+    '<rootDir>/src/.*repository.ts$',
+    '<rootDir>/src/.*guard.ts$',
+    '<rootDir>/src/api/auth/.*.ts$',
+  ],
+  coverageThreshold: {
+    global: {
+      branches: 85,
+      functions: 85,
+      lines: 85,
+      statements: 85
+    }
+  },
+  setupFiles: [
+    '<rootDir>/jest.setup.ts'
```

Comment:
`jest.setup.ts` is not generated for mongodb example app. Tests are running even without the configuration so we should probably consider removing `setupFiles` prop from e2e config on mongodb.